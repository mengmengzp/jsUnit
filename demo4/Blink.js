import React from 'react';

const Blink = () => {
    const url = 'https://www.baidu.com';
    const text = '百度一下，你就知道';
    return <a href = {url}>{text}</a>
};

export default Blink;