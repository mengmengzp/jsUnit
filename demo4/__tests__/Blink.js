import Blink from "../Blink";
import renderer from 'react-test-renderer';
import React from 'react';

test('Blink 正常渲染', () => {
    const tree = renderer.create(<Blink />).toJSON();
    expect(tree).toMatchSnapshot();
})