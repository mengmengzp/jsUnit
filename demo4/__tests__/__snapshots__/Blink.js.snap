// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blink 正常渲染 1`] = `
<a
  href="https://www.baidu.com"
>
  百度一下，你就知道
</a>
`;
